apps/openmw/mwdialogue/scripttest.cpp

```diff
@@ -84,8 +83,7 @@ void test(const MWWorld::Ptr& actor, int &compiled, int &total, const Compiler::
 
                 if (!success)
                 {
-                    Log(Debug::Warning)
-                        << "compiling failed (dialogue script)\n" << info->mResultScript << "\n\n";
+                    Log(Debug::Error) << "Error: script compiling failed (dialogue script): " << info->mResultScript;
```

Comment:
The line breaks are gone.

Stating that "script" compilation specifically was failed here and in other instances of such messages is probably redundant.

Alternatively, here it could be dealt with like "dialogue script compiling/compilation failed".

Reply:
> Stating that "script" compilation specifically was failed here and in other instances of such messages is probably redundant.

Do you suggest to remove this message at all and use the single message as in dialogue scripts?

> The line breaks are gone.

There is no much gain from them since it is the same message, not different ones. They can be misleading here.

Reply:
>Do you suggest to remove this message at all and use the single message as in dialogue scripts?

No, I meant that adding the word "script" is unnecessary, it's obvious from the context.